content/tutorials/manage-account-settings/require-destination-tags.md

```diff
@@ -1,74 +1,140 @@
+
+For example:
+
+```js
+const prepared = await api.prepareTransaction({
+  "TransactionType": "AccountSet",
+  "Account": "rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe",
+  "SetFlag": 1 // RequireDest
+})
+console.log("Prepared transaction:", prepared.txJSON)
+const max_ledger = prepared.instructions.maxLedgerVersion
+
+const signed = api.sign(prepared.txJSON, "s████████████████████████████")
+console.log("Transaction hash:", signed.id)
+const tx_id = signed.id
+const tx_blob = signed.signedTransaction
+
+const prelim_result = await api.request("submit", {"tx_blob": tx_blob})
+console.log("Preliminary result:", prelim_result)
+const min_ledger = prelim_result.validated_ledger_index
+
+// min_ledger, max_ledger, and tx_id are useful for looking up the transaction's
+// status later.
 ```
 
-<!-- MULTICODE_BLOCK_END -->
+{{ start_step("Send AccountSet") }}
+<button id="send-accountset" class="btn btn-primary previous-steps-required" data-wait-step-name="Wait">Send AccountSet</button>
+<div class="loader collapse"><img class="throbber" src="assets/img/xrp-loader-96.png">Sending...</div>
+<div class="output-area"></div>
+{{ end_step() }}
+
+
+### {{n.next()}}. Wait for Validation
+
+Most transactions are accepted into the next ledger version after they're submitted, which means it may take 4-7 seconds for a transaction's outcome to be final. If the XRP Ledger is busy or poor network connectivity delays a transaction from being relayed throughout the network, a transaction may take longer to be confirmed. (For information on how to set an expiration for transactions, see [Reliable Transaction Submission](reliable-transaction-submission.html).)
+
+{{ start_step("Wait") }}
+{% include '_snippets/interactive-tutorials/wait-step.md' %}
+{{ end_step() }}
+
+
+### {{n.next()}}. Confirm Account Settings
+
+After the transaction is validated, you can check your account's settings to confirm that the Require Destination Tag flag is enabled.
+
+```js
+let account_info = await api.request("account_info", {
+    "account": "rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe",
+    "ledger_index": "validated"
+})
+const flags = api.parseAccountFlags(account_info.account_data.Flags)
+console.log(JSON.stringify(flags, null, 2))
+// Look for "requireDestinationTag": true
+```
+
+{{ start_step("Confirm Settings") }}
+<button id="confirm-settings" class="btn btn-primary previous-steps-required" data-wait-step-name="Wait">Confirm Settings</button>
+<div class="loader collapse"><img class="throbber" src="assets/img/xrp-loader-96.png">Sending...</div>
+<div class="output-area"></div>
+{{ end_step() }}
+
+For further confirmation, you can send test transactions (from a different address) to confirm that the setting is working as you expect it to. If you a payment with a destination tag, it should succeed, and if you send one _without_ a destination tag, it should fail with the error code [`tecDST_TAG_NEEDED`](tec-codes.html).
```

Comment:
The setup involved in doing this is kind of complicated (you need another address to send from) so I opted not to provide a code sample in the document.